README.md

```diff
@@ -246,10 +246,12 @@ Some nice to have but probably separate items:
 ### Building the Runtime
 
 Contributors and other advanced users may want to build the runtime from source. To do so, you'll need to have Maven installed.
+`Java version 11+` required for a proper build. You can download and install [Java 11 Temurin](https://adoptium.net/temurin/releases/?version=11)
 
 Basic steps:
 
-* `mvn clean install` to run all of the project's tests and install the library in your local repo
+* For the 1st build you need to create `testsuite` folder manually `mkdir testsuite`
```

Comment:
This is not correct, the Maven plugin will perform the right operation.

Reply:
If I remove the `testsuite` folder and try to run `mvn clean install` it fails with:
```
[ERROR] Failed to execute goal com.dylibso.chicory:test-gen-plugin:999-SNAPSHOT:wasm-test-gen (default) on project runtime: Failed to list wast files in ~/chicory/testsuite -> [Help 1]
```

Reply:
Thanks for reporting! This is a bug/regression.
Fixed in: #313